### PR TITLE
MM-26621 Add analytic event for ask community link

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Add an analytic event for tracking `onClick` of ask the community link

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-26621

### CODE REVIEW

File: utils/constants.jsx [hunk 1]
```diff
@@ -1448,8 +1448,8 @@ export const Constants = {
     AUTOCOMPLETE_SPLIT_CHARACTERS: ['.', '-', '_'],
     ANIMATION_TIMEOUT: 1000,
     SEARCH_TIMEOUT_MILLISECONDS: 100,
-    DIAGNOSTICS_RUDDER_KEY: 'placeholder_rudder_key',
-    DIAGNOSTICS_RUDDER_DATAPLANE_URL: 'placeholder_rudder_dataplane_url',
+    DIAGNOSTICS_RUDDER_KEY: '1dodSrnEWiacrD76jx0nVToHNGu',
```
Comment (sudheerDev) [inline utils/constants.jsx:1451]: will be removed when testing is done

**Comment by jasonblais** [comment]
@sudheerDev it looks like the tests failed so the test server couldn't be generated.

@Willyfrog friendly reminder to help review this PR for today's feature complete.

**Comment by Willyfrog** [comment]
did a quick review as I'm OoO until monday and my internet is quite limited.

**Comment by jgilliam17** [comment]
@sudheerDev `Ask the community` link is not opening for me on the test server.

**Pull request merged**